cmd/suggestion/hyperopt/v1alpha3/Dockerfile

```diff
@@ -1,13 +1,26 @@
+FROM golang:alpine AS build-env
+# The GOPATH in the image is /go.
+ADD . /go/src/github.com/kubeflow/katib
+RUN if [ "$(uname -m)" = "ppc64le" ] || [ "$(uname -m)" = "aarch64" ]; then \
+        apk --update add git gcc musl-dev && \
+        go get github.com/grpc-ecosystem/grpc-health-probe && \
+        mv $GOPATH/bin/grpc-health-probe /bin/grpc_health_probe && \
+        chmod +x /bin/grpc_health_probe; \
```

Comment:
we suggest that `grpc_health_probe` should keep consistent in all kinds of platform.
Please refer to #893

Reply:
done :) shall I retest it?